schema/double.go

```diff
@@ -13,19 +43,22 @@ func writeDouble(r float64, w io.Writer) error {
 }
 `
 
+// DoubleField represents a float(64) Avro field.
 type DoubleField struct {
 	PrimitiveField
 }
 
+// NewDoubleField constructs a new `DoubleField` for the given definition
 func NewDoubleField(definition interface{}) *DoubleField {
 	return &DoubleField{PrimitiveField{
 		definition:       definition,
 		name:             "Double",
-		goType:           "float64",
+		goType:           reflect.Float64.String(),
 		serializerMethod: "writeDouble",
 	}}
 }
 
+// AddSerializer includes the required methods, structs and imports for a `DoubleField` to the given generator package.
```

Comment:
Wouldn't it make more sense to put this comment on the interface that this method is fulfilling? Having it duplicated in every type seems repetitive?

Reply:
I agree with you. I will move it to the interface once we continue refactoring the package.